contrib/cirrus/setup_environment.sh

```diff
@@ -42,8 +42,11 @@ case "${OS_REL_VER}" in
             ln -f "$CRIO_RUNC_PATH" "/usr/bin/runc"
         fi
         ;;
-    fedora-30) ;;
-    fedora-29) ;;
+    fedora-30) ;&  # continue to next item
+    fedora-29)
+        [[ "$ADD_SECOND_PARTITION" != "true" ]] || \
+            bash "$SCRIPT_BASE/add_second_partition.sh"
```

Comment:
Any reason not to use a positive test with `&&`, or an `if`? I find this type of test painful to read.

Reply:
Only that it saves typing, but yes the negative logic makes it hard to grok.  I'll fix.